internal/clients/config_status_test.go

```diff
@@ -1,26 +1,24 @@
-package dataplane_test
```

Comment:
This was specifically designed this way to be an "external" package for the SUT so that it could test it as if it didn't have access to all the unexported fields and functions.